README.md

```diff
@@ -114,6 +114,8 @@ Pubsub requires kafka to be running, and to ensure it can connect and interact w
         "value": "http://10.0.2.15:9130"
       }
 ```
+So, these parameters should be added as ENV_VARIABLES to the mod-pubsub build. Otherwise, build will fail.
```

Comment:
I think it will be enough to just substitute "should" with "must" in the sentence above the params, no need in additional message

Reply:
Done